components/infobox/wikis/stormgate/infobox_skill_custom.lua

```diff
@@ -247,4 +263,17 @@ function CustomSkill._hotkeys(hotkey1, hotkey2)
 	return Hotkeys.hotkey2(hotkey1, hotkey2, 'plus')
 end
 
+---@param patchName string
+---@return string
+function CustomSkill._getPatchName(patchName)
```

Comment:
use module:Infobox/Extension/PatchAuto instead

Reply:
restrict the patch check more using `String.endsWith(patch.pagename, '/' .. patchName)`